app/core/config.py

```diff
@@ -60,5 +60,6 @@
 NF_PIPELINE_URL = config("NF_PIPELINE_URL")
 NF_WORK_DIR = config("NF_WORK_DIR")
 SEQERA_WORKFLOW_LAUNCH_URL = config("SEQERA_WORKFLOW_LAUNCH_URL")
+SEQERA_API = config("SEQERA_API")
```

Comment:
There are two URLs for which base (HOST) will be same `SEQERA_WORKFLOW_LAUNCH_URL` and  `SEQERA_API`.
Would it be possible to just have `SEQERA_API` and remove `SEQERA_WORKFLOW_LAUNCH_URL` and define the LAUNCH_URL from `SEQERA_API`.
If not in this PR then may be in future

Reply:
I did think about it @kamaldodiya and I agree with you. I thought maybe after we get all the approvals for this PR I can make that update to reduce the noise.